app/shared/filters.js

```diff
@@ -67,7 +67,7 @@ angular.module('portainer.filters', [])
 .filter('nodestatusbadge', function () {
   'use strict';
   return function (text) {
-    if (text === 'Unhealthy') {
+    if (text == 'down' || text == 'Unhealthy') {

```

Comment:
Use `===` for string comparison.

`if (text === 'down' || text === 'Unhealthy') {`